src/routes/+layout.svelte

```diff
@@ -3,7 +3,7 @@
   import DocsMenu from "$docs/components/DocsMenu.svelte";
   import { page } from "$app/stores";
   import { goto, afterNavigate } from "$app/navigation";
-  import HeaderTitle from "../lib/components/HeaderTitle.svelte";
```

Comment:
oh, is this some new cool svelte magic?

Reply:
you mean the `$lib`? not new, I think you can do that with rollup or any bundler - i.e. we can do that in NNS-dapp too if we want too but we would need an external lib. In SvelteKit it's included and configured out of the box.

Reply:
ah, ok, let's think about it. For me it would depend more on the VS Code integration to be honest.